docs/table/access_table.rst

```diff
@@ -904,7 +893,9 @@ The examples below illustrate dealing with bytestring data in ``astropy``.
        def
 
     >>> t['a'] == 'bä'
-    array([ True, False], dtype=bool)
+    array([ True, False])
+
+.. doctest-skip::
```

Comment:
Previously this example was skipped by the `doctest-skip-all` directive at line 879, so replacing that with a `doctest-skip` here increases test coverage by allowing the preceding examples to be tested.

The example deals with I/O, so if this was to be tested it would be necessary to add setup and teardown code, and it also requires `h5py` to be installed. It certainly looks easier to skip this test.